cms/envs/common.py

```diff
@@ -114,6 +114,15 @@
     # Methods to derive settings
     _make_mako_template_dirs,
     _make_locale_paths,
+
+    # Documentation links for course apps
+    CALCULATOR_HELP_URL,
```

Comment:
There is a comment in this file that says "Warning: Do NOT add any new variables to this list. This is incompatible with future plans to
have more logical separation between LMS and Studio (CMS). It is also incompatible with the
direction documented in OEP-45: Configuring and Operating Open edX:
https://open-edx-proposals.readthedocs.io/en/latest/oep-0045-arch-ops-and-config.html"

Reply:
I've split this out again.